applications/fax/src/fax.hrl

```diff
@@ -88,9 +88,10 @@
           "-sOutputFile=~s -- ~s > /dev/null 2>&1"
           "&& echo -n success"
         >>).
--define(CONVERT_IMAGE_CMD, <<"convert -density 204x98 "
+-define(CONVERT_IMAGE_CMD, <<"convert ~s -resample 204x98 "
```

Comment:
`-density` keyword is overrides  image density in image file. Correct usage `-resample`
Also as I understand `convert` utility options input file must goes before options of output file.